src/Symfony/Component/Yaml/Tests/InlineTest.php

```diff
@@ -1004,4 +1004,10 @@ public function testParseQuotedReferenceLikeStringsInSequence()
 
         $this->assertSame(['&foo', '&bar', '&baz'], Inline::parse($yaml));
     }
+
+    public function testParseSequenceWithEmptyElement()
+    {
+        $this->assertSame(['foo', null, 'bar'], Inline::parse('[foo, , bar]'));
+        $this->assertSame([null, 'foo', 'bar'], Inline::parse('[, foo, bar]'));
```

Comment:
Should we add an assertion with `[foo, bar, ]`?

Reply:
as far as I can see `[foo, bar, ]` parses to `['foo', 'bar']` and not `['foo', 'bar', null]`.

Reply:
but we can still add a test for this indeed